charts/epinio/100.0.0+up1.0.0/charts/minio/templates/psp.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.serviceAccount.create -}}
+{{- if .Values.global.rbac.pspEnabled }}
```

Comment:
While installing helm chart version from this pr within Rancher Dashboard Airgapped throws this error:
```
helm upgrade --install=true --namespace=cattle-epinio-system --timeout=10m0s --values=/home/shell/helm/values-epinio-100.0.0-up1.0.0.yaml --version=100.0.0+up1.0.0 --wait=true epinio /home/shell/helm/epinio-100.0.0-up1.0.0.tgz
--
Tue, Aug 9 2022 11:46:02 am | Release "epinio" does not exist. Installing it now.
Tue, Aug 9 2022 11:46:02 am | Error: template: epinio/charts/minio/templates/psp.yaml:2:14: executing "epinio/charts/minio/templates/psp.yaml" at <.Values.global.rbac.pspEnabled>: nil pointer evaluating interface {}.pspEnabled
```

Reply:
this was worked-around by adding `pspEnabled:true` in Rancher Desktop
![image](https://user-images.githubusercontent.com/37271841/183647685-68d38178-7f44-45db-8462-d03070c0f44e.png)

Reply:
Fixed here: https://github.com/rancher/charts/pull/1814/commits/f40f10fef9826baa1b2e9e29f1a5d32e1603b019